Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -3985,28 +4690,29 @@
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Soil_Temperature_Sensor;1",
       "OutputDtmi": "dtmi:com:willowinc:TemperatureSensor;1"
     },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Solar_Azimuth_Angle_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:SolarAzimuthAngleSensor;1"
+    },
     {
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Solar_Radiance_Sensor;1",
       "OutputDtmi": "dtmi:com:willowinc:SolarIrradianceSensor;1"
     },
     {
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Solar_Thermal_Collector;1",
-      "IsIgnored": true,
       "OutputDtmi": "dtmi:com:willowinc:SolarThermalCollector;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:SpaceCode;1",
-      "IsIgnored": true,
-      "OutputDtmi": "dtmi:com:willowinc:SpaceCode;1"
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Solar_Zenith_Angle_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:SolarZenithAngleSensor;1"
     },
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Space_Heater;1",
-      "OutputDtmi": "dtmi:com:willowinc:UnitHeater;1"
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Space;1",
+      "OutputDtmi": "dtmi:com:willowinc:Space;1"
```

Comment:
This should be room, not space. Willow.Space is considered an abstract class, and should not be directly assigned to